### ISSUE TITLE
Copy results -> undefined result is empty

### ISSUE THREAD

**Issue by DeniseSl22** [bridgedb/Goliath#9]
Hi @nunogit ,

When I copy the results column, the unmapped IDs (given in results column as "undefined" turn into a blank object.... Why is that? Perhaps we should replace it with NAN? And then if you add these results again in Goliath, skip everything that is called NAN?

**Comment by Chris-Evelo** [comment]
NAN literally stands for Not A Number, and that is true for many valid IDs. So while I like the idea of showing them somehow and being able to skip them again on import we might have to still think about how exactly.

**Comment by DeniseSl22** [comment]
Yeah, NAN is just something I thought of very fast, but we could think of something else. Either way, it should be clear to users if they copy the results back in Goliath, which IDs are missing (not give blank lines, they do not stand out enough).
